Change to Python 3 syntax

ahijevyc opened the pull request:
Converted from python 2 to 3 syntax. Mostly print() statements.

ahijevyc committed:
Ran 2to3

rsobash reviewed (commented):
Check if python 3 changes work with python 2.7.

ahijevyc committed:
added parentheses to print statement

ahijevyc committed:
removed reference to overlay

ahijevyc committed:
Maybe remake pickle files; use python 2 or 3.

Fixed tab, pickle read, no overplot

Replaced tabs with spaces.

Added binary flag to pickle read ('r' to 'rb')--for python3.

Removed reference to 'over' option (for overplot).

ahijevyc commented:
In order to test with Python 2, had to change a few things.

- Remade pickle files
- Removed reference to overlay option
- replaced tabs with spaces
- absolute path to pngquant in Ryan Sobash's home directory

One more change related to Python 3.
added 'b' binary flag to pickle read

Once pickle files are made for either Python 2 or Python 3, script should run with either version.

Tested with python 2 and these modules on cheyenne3:
Currently Loaded Modules:
  1) ncarenv/1.2    3) ncarcompilers/0.4.1   5) vim/8.0         7) numpy/1.13.3       9) scipy/0.19.1  11) netcdf4-python/1.2.7
  2) intel/17.0.1   4) mpt/2.19              6) python/2.7.13   8) matplotlib/2.0.2  10) netcdf/4.6.1

ahijevyc committed:
binary flag for pickle dump
